RecoBTag/ChargeTagging/src/CandidateChargeBTagComputer.cc

```diff
@@ -0,0 +1,228 @@
+#include "RecoBTag/ChargeTagging/interface/CandidateChargeBTagComputer.h"
+
+CandidateChargeBTagComputer::CandidateChargeBTagComputer(const edm::ParameterSet & parameters):
+  useCondDB_(parameters.existsAs<bool>("useCondDB") ? parameters.getParameter<bool>("useCondDB") : false),
+  gbrForestLabel_(parameters.existsAs<std::string>("gbrForestLabel") ? parameters.getParameter<std::string>("gbrForestLabel") : ""),
+  weightFile_(parameters.existsAs<edm::FileInPath>("weightFile") ? parameters.getParameter<edm::FileInPath>("weightFile") : edm::FileInPath()),
+  useGBRForest_(parameters.existsAs<bool>("useGBRForest") ? parameters.getParameter<bool>("useGBRForest") : false),
```

Comment:
Do you mean by setting the default option useGBRForest_ "true" or by removing the parameter and fixing it to "true"?
Default changed to "true"